packages/server/router/index.ts

```diff
@@ -270,6 +270,7 @@ export default function buildBaseApi(server: Server) {
         'POST',
         (req, res) => {
             const body: ISettings[] | Error = JsonSafeParse(
+                false,
```

Comment:
Move, this boolean as second argument, or move this statement into named args like

```ts
const JsonSafeParse(params: { payload: string, isFile: boolean, onError: Error})
```